app/src/main/java/ip/team13/petowner/data/dto/AuthResponse.kt

```diff
@@ -0,0 +1,10 @@
+package ip.team13.petowner.data.dto
+
+import com.squareup.moshi.Json
+
+data class AuthResponse(
```

Comment:
I think we decided earlier that all DTO models should have a `Model` suffix in the name, while the rest of domain models should have a name without `Model` suffix.